packages/koa-metrics/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/shopify/quilt/blob/master/packages/koa-metrics/README.md",
   "dependencies": {
-    "hot-shots": "^5.4.1",
+    "hot-shots": "^6.1.1",
```

Comment:
We generally update our dependencies using `yarn` so the corresponding `yarn.lock`s resolve properly. 

```
yarn add hot-shots
```

Reply:
I didn't include the root `yarn.lock` when working on `koa-metrics`. Should be fixed now.